components/meter/meter-radial.js

```diff
@@ -0,0 +1,52 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { heading4Styles } from '../typography/styles.js';
+
+class MeterRadial extends LitElement {
+	static get styles() {
+		return [ heading4Styles, css`
+		:host {
+			display: block;
+		}
+		.d2l-meter-radial-full-bar,
+		.d2l-meter-radial-progress-bar {
+			fill: none;
+			stroke-linecap: round;
+			stroke-width: 9;
+		}
+		.d2l-meter-radial-full-bar {
+			stroke: var(--d2l-color-gypsum);
```

Comment:
Don't forget to import colors if you're using them.

Reply:
I think this is working because typography imports colours, but that isn't guaranteed.